frontend/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -80,12 +90,13 @@ class GraphVizChart extends React.PureComponent<Props> {
 
   public render = (): React.ReactNode => {
     const width: number = this.props.element.get("width") || this.props.width
-
+    const height: number =
```

Comment:
Shouldn't this use the same logic as the code block above for `// Override or reset the graph height`?

Reply:
Is the same logic, this.props.element.get("height") have preference

Reply:
Above also uses `this.originalHeight` .. should it be a possible value for `height` here as well?